interface/modules/zend_modules/public/js/multipledb/jquery.validate.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery Validation Plugin - v1.15.0 - 2/24/2016

```

Comment:
done, i use in this 
public/assets/jquery-validation-1-13-0/dist/jquery.validate.min.js